src/app/tests/suites/certification/Test_TC_IDM_2_2.yaml

```diff
@@ -37,9 +37,10 @@ tests:
           [[Endpoint = Endpoint1, Cluster = ClusterID]] On receipt of this
           message, DUT should send a report data action with the attribute value
           to the DUT."
-      verification:
-          "In case of chip tool, here is an example command to use sudo
-          ./chip-tool any read-by-id 0x3 0xFFFFFFFF 1 0"
+      verification: |
+          In case of chip tool, here is an example command to use
+
+          sudo ./chip-tool any read-by-id 0x3 0xFFFFFFFF 1 0
```

Comment:
@kowsisoundhar12 can you please respond to  @markus-becker-tridonic-com ?

Reply:
@kowsisoundhar12 This was about the `sudo` comment :)

Reply:
@cjandhyala can u please check above comments ,I have generated the Manual scripts using Following gsheet so all the verification steps are based on the gsheet ,please let us know your thoughts on this 

https://docs.google.com/spreadsheets/d/1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY/edit#gid=1886169525

Reply:
@markus-becker-tridonic-com @woody-apple for now we are just going verbatim as per the verification steps to avoid having 2 sources of truth. If the verification steps change , we will also then update the scripts accordingly .

Reply:
Why are the verification steps using sudo?  They should not be using sudo.

Reply:
@cjandhyala can you please comment on the reason behind using sudo in verification steps ? Can it be fixed please ?